src/nft-card-front.ts

```diff
@@ -202,24 +222,15 @@ export class NftCardFrontTemplate extends LitElement {
   }
 
   public getAssetPriceTemplate() {
-    // TODO: Needs to account for tokens with images not symbols
-    // If payment_token.image_url then use token image instead of symbol
-    let prevPriceTemplate: TemplateResult = html``
-    let currentPriceTemplate: TemplateResult = html``
-
-    if (this.lastSaleData?.paymentToken) {
-      // const currentPriceSymbol = this.lastSaleData.paymentToken.symbol === 'ETH' ? 'Ξ ' : ''
-      currentPriceTemplate = this.getCurrentPriceTemplate(this.lastSaleData.paymentToken, this.lastSaleData.currentPrice)
 
-    }
+    const currentPriceTemplate = this.lastSaleData?.paymentToken ?
+        NftCardFrontTemplate.getPriceTemplate(PriceType.Current, this.lastSaleData.paymentToken,
+            this.lastSaleData.currentPrice) : null
 
-    if (this.asset?.lastSale) {
-      // @ts-ignore ignore until LastSale type gets added to SDK
-      const formattedPrevPrice = this.asset.lastSale.total_price / Math.pow(10, this.asset.lastSale.payment_token.decimals)
-      // @ts-ignore ignore until LastSale type gets added to SDK
-      const prevPriceSymbol = this.asset.lastSale.payment_token.symbol === 'ETH' ? 'Ξ ' : ''
-      prevPriceTemplate = html`<div class="asset-detail-price-previous">Prev. ${prevPriceSymbol} ${formattedPrevPrice}</div>`
-    }
+    const prevPriceTemplate = this.lastSaleData?.paymentToken ?
```

Comment:
why aren't we checking `this.asset?`'s paymentToken and using it? seems like there's two ways to check the lastSale here. is lastSaleData no longer suppsed to be used